docs/process-services-cloud/directives/claim-task-cloud.directive.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: Claim Task Cloud Directive
+Added: v3.9.0
+Status: Experimental
+Last reviewed: 2020-06-09
+---
+
+# [Claim task Cloud directive](../../../lib/process-services-cloud/src/lib/task/directives/claim-task-cloud.directive.ts "Defined in claim-task-cloud.directive.ts")
+
+Claims a task
+
+## Basic Usage
+
+```html
+<button adf-cloud-claim-task [appName]="appName" [taskId]="taskId" (success)="onTaskClaimed()">Claim</button>
+```
+
+## Class members
+
+### Properties
+
+| Name | Type | Default value | Description |
+| ---- | ---- | ------------- | ----------- |
+| appName | `string` | " " | (Required) The name of the application. |
+| taskId | `string` | " " | (Required) The id of the task. |
```

Comment:
This has to be double quotes without space. Like this `""`. Please update all the occurrences